### ISSUE TITLE
user feedback on time and date display

### ISSUE THREAD

**Issue by georgiamshaw** [fac-17/week-3-MEIA#22]
<img width="432" alt="Screenshot 2019-07-19 at 10 21 25" src="https://user-images.githubusercontent.com/36238916/61524793-1bd9ca00-aa0f-11e9-901e-e53bcb22470a.png">

I know it says morning above it, but maybe it would be a good idea to have am or pm after the time? As a user I would expect to see it there. Unless your clock is 24 hours in which case obviously you don't need it - but it just feels like something is missing from the time. 
Also - if possible it might be nice to show the day as well? For example, Friday, 19th July instead of 19/07/2019

**Comment by Dalmano** [comment]
I agree. if you think of it from an accessibility point of view(even though it is 100%, dayyum btw) it could be a nice touch to make it apparent if it is am or pm. And the same suggestion for the day as well. A vision-impaired user would really benefit from such things

**Comment by georgiamshaw** [comment]
Just seen in your code that you are using a 24 hour clock so fair enough - but perhaps it can be a tiny bit clearer to the user.

**Comment by crianonim** [comment]
Yes we went for a 24 hr display. It's supposed to be 'railway time' and I think in the uk time on timetables and such are display in 24 hr format.

But thank you for your feedback, it clearly needs some more thinking.
